README.md

```diff
@@ -1,2 +1,70 @@
-# hello-world-rails-api
-Hello world rails API
+# Hello World Rails React
+
+> A simple RAILS Api with one end point that returns a random greeting from a Postgres Database. It returns JSON. 
+
+> The front end is in this [repo](https://github.com/Beardless-sheik/hello-world-react-front-end)
+
```

Comment:
in your frontend repo there is not link to the backend repo please add this 😸

Reply:
Thanks for the observation @alexander16108  . Link added